### ISSUE TITLE
Remove warnings about `cfg_target_feature`

### ISSUE THREAD

**Issue by hdevalence** [dalek-cryptography/curve25519-dalek#204]
Compiling `curve25519-dalek` gives
```
warning: the feature `cfg_target_feature` has been stable since 1.27.0 and no longer requires an attribute to enable
```
We should just support the latest stable version and use `cfg_target_feature` unconditionally.

**Comment by amanjeev** [comment]
I can try and fix this. cc @isislovecruft

**Comment by DokFaust** [comment]
Hi everyone! :)
Both with the latest `nightly` and `stable` I am not able to reproduce the warning, either with nightly or stable toolchain. Btw I am not able to find the tracking issue for `cfg_target_feature` as it should have been stabilized recently.

**Comment by hdevalence** [comment]
This was closed by #215
